README.md

```diff
@@ -41,15 +40,10 @@ $ juju relate kube-state-metrics prometheus-k8s
 In a similar manner any charm that exposes a scrape target may be
 related to the Prometheus charm.
 
-At present it is expected that all relations the Prometheus Operator
-partakes in are within the same Juju model.
-[Further development](https://github.com/canonical/prometheus-operator/issues/58)
-may extend this to allow cross model scrape targets.
```

Comment:
This is still the case. Prometheus might in some cases (like same-cluster, different models) be able to scrape across model boundaries, but usually what we recommend is for users to set up a grafana agent in the remote model and use `remote_write` to get their metrics into Prometheus.

The reference to the issue can go, however, as we're not expecting to change this behavior.

Reply:
@simskij 
what is about adding such a note?


> Note: At present it is expected that all relations the Prometheus Operator 
> partakes in are within the same Juju model. For alternative, please set up a 
> grafana agent in the remote model and use `remote_write` to get metrics into Prometheus.

Reply:
> At present it is expected that all relations that Prometheus are part of are within the same Juju model. For cross-model relations, please set up a Grafana Agent in the remote model and use remote_write to get the metrics into Prometheus.

some minor grammar changes, but all in all - looks good! 👍🏼

Reply:
done